README.md

```diff
@@ -1,142 +1,232 @@
+[![Twitter](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=wso2)
+
+# WSO2 Enterprise Integrator
+
+WSO2 Enterprise Integrator is an open source, fast, cloud native and scalable integration solution that is the core of 
+WSO2 Integration Agile Platform. It enables enterprise integration developers to build sophisticated integration 
+solutions to achieve digital agility. As a mature integration product since 2005,*(branded as WSO2 ESB at the time)*, it 
+continues to be the most sophisticated and extensible open source enterprise integration solution available.
+
+Actively maintained, with commercial support from WSO2 Inc, WSO2 Enterprise Integrator is widely used in production at 
+companies around the globe starting from startups to fortune 500 companies in the fields of government, healthcare, 
+banking, education, communication, etc.
+
+[Installation](https://docs.wso2.com/display/EI640/Installation+Guide) | 
+[Documentation](https://docs.wso2.com/display/EI640/) | 
+[Mailing Lists](https://wso2.com/mail/) | 
+[Blog](https://wso2.com/blogs/thesource/) | 
+[Support](https://wso2.com/subscription) | 
+[Nightly Builds](https://wso2.org/jenkins/view/products/job/products/job/product-ei/)
+
+
+## Outline 
+
+- [**Why WSO2 Enterprise Integrator?**](#Why-WSO2-Enterprise-Integrator)
+- [**Features**](#Features)
+- [**Distributions**](#Distributions)
+- [**Installation and Run**](#Installation-and-Run)
+- [**Artifact Development**](#Artifact-Development)
+- [**Building from Source**](#Building-from-Source)
+- [**Enterprise Support**](#Enterprise-Support)
+- [**Licence**](#License)
+
+
+## Why WSO2 Enterprise Integrator
+
+WSO2 Enterprise Integrator is an open source, hybrid integration platform. It is comprised of profiles that address different
+parts of a complete integration story. 
+
+**Enterprise Service Bus profile**: If you are trying to interconnect your enterprise applications on-premise, legacy 
+  or cloud, WSO2 EI can act as a Service Bus. It can help transforming messages 
+  to different formats, standards, use different protocols to communicate and mediating messages across the applications. 
+  With 100+ ready made cloud connectors, intuitive tooling,  analytic capabilities WSO2 EI provides a greater agility to 
+  meet growing and changing enterprise demands. It can also accelerate bringing your enterprise data to the screen by 
+  exposing them as services, APIs.
+ 
+**Message Broker profile**: This allows for queueing of messages (using the AMQP protocol) where you do not require an 
+  immediate response to continue processing. WSO2 EI acts as a Message Broker server.  
+
+**Business-process profile**: If your integration story contains human interactions (i.e., approval process) WSO2 EI  
+  provides BPS/BPMN and human task capabilities to develop work-flows.
```

Comment:
We need add requirement of "statefull integration/orchestration" BPEL,BPMN are not only for human interactions

Reply:
Updated as 

```
**Business-process profile**: If your integration story contains human interactions (i.e., approval process) or stateful 
  integration/orchestration, WSO2 EI provides BPEL/BPMN and human task capabilities to develop work-flows.
```